duckduckgo/iOS
# Autofill display "Suggested logins" for current site above main logins list

Task/Issue URL: https://app.asana.com/0/0/1203313768964239/f
Tech Design URL:
CC: @THISISDINOSAUR 

**Description**:
Automatically display logins for current site as "Suggested Logins" above main list. Note that this is subject to copy review. Work to reduce auth friction in multi-stage login processes will be targeted in a separate PR (https://app.asana.com/0/1201462886803403/1203091315317194/f)



**Steps to test this PR**:
1. Save a login for a specific site. Open that site in a tab
2. Open Autofill via the browsing menu
3. Confirm that the login saved for this site is displayed in a separate section at the top called "SUGGESTED"
4. Go to another site without a saved login
5. Open Autofill via the browsing menu
6. Confirm that there is no Suggested section at the top of the logins list
7. Repeat steps 1 and 4 but access Autofill via the Settings menu and confirm that there is no Suggested section at the top of the logins list in either case



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
